weapons.yaml

```diff
@@ -33,4 +33,22 @@
 
     - name: Copy .zshrc template
       command: cp ~/.oh-my-zsh/templates/zshrc.zsh-template ~/.zshrc
```

Comment:
I would suggest that the template be kept in the repository and copied with the playbook using `copy` or `template` modules of Ansible. Using the `command` module will probably cause [ansible-lint](https://github.com/willthames/ansible-lint) to barf.

Reply:
I don't want to parametrize the whole file for the time being, I don't plan to add much more options, and it's easier for the time being to just update the latest version from the oh-my-zsh repo